rollup.config.executor.js

```diff
@@ -0,0 +1,79 @@
+//import commonjs from '@rollup/plugin-commonjs';
+//import hash from 'rollup-plugin-hash';
+import outputManifest from 'rollup-plugin-output-manifest'
+import typescript from '@rollup/plugin-typescript';
+
+
+var outDir = 'build/resources/main/assets/react4xp';
+
+export default {
+	external: [
+		//'prop-types',
+		'react',
+		'react-dom'
```

Comment:
Not synced with the pluggable externals setup, but that's maybe not necessary for a separate executor?

Reply:
Yeah, that config really doesn't matter, the file doesn't import anything.